tools/cloud-build/hpc-toolkit-pr-validation.yaml

```diff
@@ -15,6 +15,9 @@
 ---
 
 steps:
+- id: git-fetch-unshallow
```

Comment:
I think we need to add this step to the `waitFor` block of `make-tests` on line 41. Right now it is not failing because `pre-commit-setup` takes longer than `git-fetch-unshallow` but it would be correct to have the explicit dependency. In this case you would have two waitFor entries ([example](https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/f4ed7c1ca864c77ea8cc21bd9286b0c48d28db95/tools/cloud-build/daily-tests/integration-group-2.yaml#L49-L51)).

Reply:
Good catch, thanks!